docs/repositories-configure/codacy-configuration-file.md

```diff
@@ -61,7 +61,7 @@ To use a Codacy configuration file:
 
 ## Syntax for ignoring files
 
-To ignore files, you must use the [Java glob syntax](https://docs.oracle.com/javase/7/docs/api/java/nio/file/FileSystem.html#getPathMatcher%28java.lang.String%29) to define one or more `exclude_paths` patterns. For example:
+To ignore files using a Codacy configuration file, you must define one or more patterns under `exclude_paths` using the [Java glob syntax](https://docs.oracle.com/javase/7/docs/api/java/nio/file/FileSystem.html#getPathMatcher%28java.lang.String%29):
```

Comment:
Here I mention the Codacy configuration file for users who land on the page using one of the new links.